research/02-repl.tex

```diff
@@ -1,5 +1,172 @@
 \section{Read-Eval-Print Loops}
-\label{sec:read-eval-print}
+\label{sec:repl}
+
+% TODO: mention that REPLs do not necessarily have to be interactive?
+Many programming languages come with an interactive environment. This
+interactive environment is an interface to the programming language's execution
+engine. These interactive environments know many different names, including but
+not limited to \textit{language shell}, \textit{command-line interpreter} and
+\textit{REPL}. There are subtle differences between these interpretations.
+These are, however, mostly of semantic value. In this report the term
```

Comment:
This is resolved, as per [line 52](https://github.com/spoofax-shell/doc/pull/9/files#diff-eb876ecf7899fba0acd8fc761ecdf982R52)